lib/rspec/core/memoized_helpers.rb

```diff
@@ -307,9 +307,13 @@ def let(name, &block)
           # We have to pass the block directly to `define_method` to
           # allow it to use method constructs like `super` and `return`.
           raise "#let or #subject called without a block" if block.nil?
-          raise(
-            "#let or #subject called with a reserved name #initialize"
-          ) if :initialize == name
+
+          # A list of reserved words that can't be used as a name for a memoized helper
+          # Matches for both symbols and passed strings
+          if [:initialize, :to_s].include?(name.to_sym)
+            raise ArgumentError.new "#let or #subject called with reserved name `#{name}`"
```

Comment:
```

lib/rspec/core/memoized_helpers.rb:314:13: C: Style/RaiseArgs: Provide an exception class and message as arguments to raise.
            raise ArgumentError.new "#let or #subject called with reserved name `#{name}`"
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
I guess RuboCop suggests writing this as:
```ruby
raise ArgumentError, "#let or #subject called with reserved name `#{name}`"
```

Reply:
Sorry for that ... should have checked again after adding the ArgumentError 🤦🏻